.github/workflows/build.yml

```diff
@@ -345,19 +345,59 @@ jobs:
       - id: printtag
         name: Output image name and tag
         run: echo "IMAGE_TAG=$(make -e print-docker-tag)" >> $GITHUB_OUTPUT
-  openshift_preflight:
-    name: Run the OpenShift Preflight check on the published images
-    needs:
-      - package_and_publish
-    runs-on: ubuntu-latest
-    env:
-      IMAGE_TAG: ${{ needs.package_and_publish.outputs.IMAGE_TAG }}
+
+  build_stackablectl:
+    name: Build stackablectl for ${{ matrix.target }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+          # Currently Widnows build fails with
+          # cargo:warning=# runtime/cgo
+          # cargo:warning=cgo: C compiler "C:\\Program" not found: exec: "C:\\Program": file does not exist
+          #   error occurred: Command "go" "build" "-buildmode" "c-archive" "-o" "D:\\a\\stackable-cockpit\\stackable-cockpit\\target\\debug\\build\\helm-sys-cce901864d7b6e8c\\out\\libgo-helm-wrapper.a" "go-helm-wrapper/main.go" with args "go-helm-wrapper" did not execute successfully (status code exit code: 1).
+          # so that Windows build is disabled for now
+          # - target: x86_64-pc-windows-gnu
+          #   os: windows-latest
+          - target: x86_64-apple-darwin
+            os: macos-latest
+          - target: aarch64-apple-darwin
+            os: macos-latest
     steps:
-      - name: Install preflight
-        run: |
-          wget https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/latest/download/preflight-linux-amd64
-          chmod +x preflight-linux-amd64
-      - name: Check container
-        run: ./preflight-linux-amd64 check container "$IMAGE_TAG" > preflight.out
-      - name: "Passed?"
-        run: '[ "$(./preflight-linux-amd64 check container "$IMAGE_TAG" | jq -r .passed)" == true ]'
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          submodules: recursive
+      - uses: actions-rs/toolchain@v1
```

Comment:
We should instead use `dtolnay/rust-toolchain` because we use it for every other pipeline as well.

Reply:
Switched in https://github.com/sbernauer/stackable-cockpit/commit/2c1d80386f21f33afc4a21e09d65f6dc6dc0e6b3